scio-examples/src/main/scala/com/spotify/scio/examples/extra/ParquetExample.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2019 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// Example: Read and Write specific and generic Avro records
+// Usage:
+
+// `sbt "runMain com.spotify.scio.examples.extra.ParquetExample
+// --project=[PROJECT] --runner=DataflowRunner --zone=[ZONE]
+// --input=[INPUT].parquet --output=[OUTPUT].parquet --method=[METHOD]"`
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio._
+import com.spotify.scio.parquet.avro._
+import com.spotify.scio.parquet.types._
+import com.spotify.scio.avro.Account
+import com.spotify.scio.io.ClosedTap
+import org.apache.hadoop.conf.Configuration
+import org.apache.avro.generic.GenericRecord
+
+object ParquetExample {
```

Comment:
looks pretty good! I think it would be good to have another job configuration that only writes Parquet data. (for example, `AvroExample` lets the user specify whether they want to read or write; `SortMergeBucketExample` has different jobs for writing, and reading, SMB data.) the idea is that the user could actually directly run those jobs on the runner of their choice, not just in a unit test! since the jobs in your ParquetExample all read Parquet input and write Parquet output, there’s not a clear way for a user to test it out themselves unless they have pre-written Parquet data lying around :slightly_smiling_face: